FlowCrypt/Controllers/Setup/SetupGenerateKeyViewController.swift

```diff
@@ -22,16 +22,16 @@ final class SetupGenerateKeyViewController: SetupCreatePassphraseAbstractViewCon
     private let service: Service
 
     init(
-        appContext: AppContext,
-        user: UserId,
+        appContext: AppContextWithUser,
+        userId: UserId,
         decorator: SetupViewDecorator = SetupViewDecorator()
     ) {
         self.attester = AttesterApi(
             clientConfiguration: appContext.clientConfigurationService.getSaved(for: user.email)
         )
         self.service = Service(
             appContext: appContext,
-            user: user,
+            userId: user,
```

Comment:
Todo - `UserId` should be retrieved from `appContext.user.userId`. Applies to all setup controllers.